test/tool-option-parsing.py

```diff
@@ -223,9 +225,14 @@ def test_compile_keymap_args(self):
                 self.xkbcli_compile_keymap.run_command_success(args)
 
     def test_compile_keymap_rmlvo(self):
-        for rmlvo in rmlvos:
+        def run(rmlvo):
             with self.subTest(rmlvo=rmlvo):
```

Comment:
`subTest` is not thread safe, so it's probably better to move the `subTest` to the `future.result()`